docs/contribution/30-overview/3-concepts/5-frontend-fragment.mdx

```diff
@@ -0,0 +1,108 @@
+
+Session-Tokens always include a session ID that is stable for the current session.
+This ID can be used to identify a user’s session and, for example, to map backend sessions.
+Session cookies cannot be used, as this does not work in WebKit-based browsers like Safari.
+
+A frontend fragment must always include the Session-Token when making requests to its backend.
+The JWT claims in the Session-Token include a key called `publicKeySerial`, which refers to the serial number of the mStudio public key used to sign the token.
+The Ed25519 public key can be retrieved from the mStudio API using the operation <OperationLink tag="Marketplace" operation="extension-get-public-key" />.
+To do this, use the publicKeySerial from the session token as the serial and the purpose `session_token`.
+
+To simplify signature verification, the public key can be retrieved in SPKI format, ASN.1 serialized, and PEM encoded via the `format` query parameter.
+
+The backend must validate the session token against the mStudio public key indicated in the token with every request,
+since the token serves as the user’s authentication credential.
+This involves decoding the JWT, retrieving the public key via the API, and validating the JWT against the key.
+A public key with a specific serial is stable and can be cached or persisted.
+
+The session token can also be used in combination with an extension secret to obtain an access token.
+For this, the operation <OperationLink tag="Marketplace" operation="extension-authenticate-with-session-token" /> can be used.
+The resulting access token is also only valid for 60 seconds and should not be persisted.
+
+The entire authentication process is illustrated in the following sequence diagram:
+
+```mermaid
+sequenceDiagram
+  participant mStudio
+  participant ExtensionF as Extension Frontend
+  participant ExtensionB as Extension Backend
+
+  mStudio ->> ExtensionF: render Frontend Fragment
+  ExtensionF ->> mStudio: get Session-Token
+  mStudio -->> ExtensionF: Session-Token
+  ExtensionF ->> ExtensionB: Request with Session-Token Header
+  ExtensionB ->> ExtensionB: Decode Session-Token
+  opt Retrieve Public Key if unknown
+    ExtensionB ->> mStudio: Get Public Key By Serial
+    mStudio -->> ExtensionB: Public Key
+  end
+  ExtensionB ->> ExtensionB: Validate Session-Token
+  ExtensionB ->> mStudio: Authenticate with Session-Token and Extension Secret
+  mStudio -->> ExtensionB: Access Token
+  ExtensionB ->> mStudio: Domain Action
+```
+
+## Kontext Parameter
+
+## Extension Bridge
+
+The Extension Bridge is a JavaScript library that can be used in frontend fragments to communicate with mStudio. It provides functions to retrieve context parameters and to obtain, decode, and validate session tokens.
+
+For more information, see the [Extension Bridge documentation](https://github.com/mittwald/flow/tree/main/packages/ext-bridge)
```

Comment:
That link isn't really helpful at the moment. Is there more documentation planned?